server/schedule/operator/builder.go

```diff
@@ -138,7 +138,7 @@ func NewBuilder(desc string, ci ClusterInformer, region *core.RegionInfo, opts .
 
 	// placement rules
 	var rules []*placement.Rule
-	if err == nil && b.GetOpts().IsPlacementRulesEnabled() {
+	if err == nil && !b.skipCorrectnessCheck && b.GetOpts().IsPlacementRulesEnabled() {
```

Comment:
If a reigon needs to be split, should we allow it to transfer to the leader

Reply:
It should only be valid for `CreateLeaveJointStateOperator` and `CreateForceTransferLeaderOperator`. Now I use two flags instead.